README.md

```diff
@@ -73,145 +72,101 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 ### Hosts File Editor
 
-- Added a scrollbar to the additional lines dialog. Thanks [@davidegiacometti](https://github.com/davidegiacometti)! (This was a hotfix for 0.64)
-- Updated the plus icon. Thanks [@niels9001](https://github.com/niels9001)! (This was a hotfix for 0.64)
-- Prevent the new entry content dialog from overlapping the title bar.
-- Updated the name for the additional lines feature. Thanks [@htcfreek](https://github.com/htcfreek)!
-- Added a workaround for an issue causing the context menu not opening on right-click. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
-
-### Image Resizer
-- Fixed a silent crash when trying to show the tier 1 context menu on Windows 11.
+- Added a warning when duplicated entries are detected. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 
 ### PowerToys Run
 
-- Added pinyin support to the search. Thanks [@frg2089](https://github.com/frg2089)!
-- Fixed an error in the TimeZone plugin preventing searching for standard time zones. Thanks [@Tantalus13A98B5F](https://github.com/Tantalus13A98B5F)!
-- Added the English abbreviations as fallbacks in the UnitConverter plugin. Thanks [@Tantalus13A98B5F](https://github.com/Tantalus13A98B5F)!
+- Support drag and dropping for file results. Thanks [@daniel-richter](https://github.com/daniel-richter)!
 
 ### Quick Accent
 
-- Added mappings for the mu, omicron, upsilon and thorn characters.
-- Added a setting to exclude apps from activating Quick Accent.
-- Fixed an issue causing the selector to trigger when leaving the lock screen. Thanks [@damienleroy](https://github.com/damienleroy)!
-- Added the Croatian, Netherlands, Swedish and Welsh character sets. Thanks [@damienleroy](https://github.com/damienleroy)!
-- Added support for more unicode characters. Thanks [@char-46](https://github.com/char-46)!
-- Shift-space can now navigate backwards in the selector. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
-- Added the Catalan accented characters. Thanks [@ivocarbajo](https://github.com/ivocarbajo) and [@codingneko](https://github.com/codingneko)!
-- Added the Kurdish accented characters.
-- Added the Serbian accented characters. Thanks [@damienleroy](https://github.com/damienleroy)!
-- Added the Irish and Scottish accented characters.
-- Added the description for the currently selected character in the selector.
-- Fixed a bug causing the selector window to appear blank.
-
-### Runner
-
-- Fixed a crash on a racing condition accessing the IPC communication with Settings.
+- Added support for dark theme. Thanks [@niels9001](https://github.com/niels9001)!
+- Increased default input delay to improve out of the box experience.
+- Fixed a bug causing the first character to not be selected when opening the overlay. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Fixed the positioning of the overlay when showing in the diagonal corners.
```

Comment:
in the what?

Reply:
You're right. Just changed it to "when showing near the horizontal edges of the screen."